ansible/playbooks/nagios/play_setup_server.yml

```diff
@@ -2,7 +2,7 @@
 - name: Install Nagios
   hosts: Nagios_Server
   vars_files:
-    - secrets_setup_server.enc
+    # - secrets_setup_server.enc
```

Comment:
commented out code?

Reply:
Yes, in order to stop the linter complaining, I need to remove the encrypted ansible vault, secrets_setup_server.enc .. however I want to leave the reference to it, and have it refer to the document.. I'll expand the comments to cover reading the docs (which details creating the file, before running the playbook ).